driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -930,6 +933,214 @@ public Date deserialize(ByteBuffer bytes) {
         }
     }
 
+    static class SimpleDateCodec extends TypeCodec<DateWithoutTime> {
+
+        public static final SimpleDateCodec instance = new SimpleDateCodec();
+        private static final Pattern IS_LONG_PATTERN = Pattern.compile("^-?\\d+$");
+
+        private static final String[] patterns = new String[] {
+                                                              "yyyy-MM-dd",
+                                                              "yyyy-MM-ddZ"
```

Comment:
To be honest, I don't remember. Probably a leftover of copying from `DateCodec`.